generator/test/good_test_data/all_types.expected

```diff
@@ -0,0 +1,232 @@
+// **************************************************************************
```

Comment:
This is not strictly related to this PR, but I saw we don't validate the outputs of the `all_types.dart` test, which wasn't great as it would only validate that it doesn't error out, but not that it produces meaningful results. I wanted to verify the outputs for the new type, so I had to add the entire file.